### PR TITLE
add layout changes for view evaluation administration 

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [View Eval Admin Issues and Enhancements #112](https://github.com/nerubia/kh360-react/issues/112)

### CODE REVIEW

**Review by Dev-Nino — APPROVED** [review]

File: src/components/badge/Badge.tsx [hunk 1]
```diff
@@ -24,6 +24,9 @@ const badge = cva(
         gray: ["bg-gray-500", "text-white"],
         greenOutline: ["border", "border-green-500", "text-green-500"],
       },
+      size: {
```
Comment (jatnerubia) [inline src/components/badge/Badge.tsx:27]: I think you need to make a `small` size too

File: src/components/badge/Badge.tsx [hunk 2]
```diff
@@ -24,6 +24,9 @@ const badge = cva(
...
+        medium: ["text-base", "px-4", "py-1.5"],
+      },
     },
     defaultVariants: {
       variant: "primary",
```
Comment (jatnerubia) [inline src/components/badge/Badge.tsx:32]: and add default size `medium` here

File: src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx [hunk 3]
```diff
@@ -31,21 +34,35 @@ export const ViewEvaluationHeader = () => {
...
-              {evaluation_administration?.name}
-            </h1>
-            <div>
-              Evaluation Period (
-              {formatDate(evaluation_administration?.eval_period_start_date)} to{" "}
-              {formatDate(evaluation_administration?.eval_period_start_date)})
+            <div className='flex gap-4 primary-outline items-end mb-4'>
+              <PageTitle>{evaluation_administration?.name}</PageTitle>
+              <Badge
+                size='medium'
```
Comment (jatnerubia) [inline src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx:43]: remove this after you set the default size in badge

Comment (jatnerubia) [reply src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx:43]: but I think you need to update other components that are using `small` badge

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/components/badge/Badge.tsx (hunk 2 above)
Comment (emmaacha) [reply src/components/badge/Badge.tsx:32]: Should the default size be `medium` kuys or `small`?

File: src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx (hunk 3 above)
Comment (emmaacha) [reply src/features/admin/evaluation-administrations/[id]/ViewEvaluationHeader.tsx:43]: Ahh so set the default to `medium`, then add a size variant for `small `badges?

File: src/components/badge/Badge.tsx (hunk 2 above)
Comment (jatnerubia) [reply src/components/badge/Badge.tsx:32]: medium 🤔

**Review by jatnerubia — APPROVED** [review]
Nice

**Pull request merged**